src/NHibernate.Test/Criteria/ReadonlyTests/QueryOverCacheableTests.cs

```diff
@@ -0,0 +1,286 @@
+using System.Linq;
+using NHibernate.Cfg;
+using NHibernate.DomainModel.Northwind.Entities;
+using NHibernate.SqlCommand;
+using NUnit.Framework;
+
+namespace NHibernate.Test.Criteria.ReadonlyTests
+{
+	[TestFixture]
+	public class QueryOverCacheableTests : CriteriaNorthwindReadonlyTestCase
+	{
+		//Just for discoverability
+		private class CriteriaCacheableTest{}
```

Comment:
To make it appear in search when user enters Criteria trying to find some Criteria related tests

Reply:
Can-it be just a comment, not a dummy class?

Reply:
>Can-it be just a comment, not a dummy class?

When I'm not sure which test fixture I need I often try to find fixture class I need by some keywords. So comment is less discoverable than dummy class. Well if you really dislike it  - you can drop it. 

>  It is there just because query-over is an added layer to criteria and so test it by the way, isn't-it?

Yes.